app/handlers/tests/test_handlers_connector.py

```diff
@@ -653,6 +707,6 @@ def test_pod_labels(self, get_connector_and_crd, mock_tenant_url, mock_image):
         pod_metadata = dep.spec["template"]["metadata"]
 
         assert pod_metadata["labels"] == labels | {
-            "app.kubernetes.io/name": "TwingateConnector",
-            "app.kubernetes.io/instance": crd.metadata.name,
+            "connector.twingate.com/name": crd.metadata.name,
+            "env": "test",
```

Comment:
Do we need this `env` label? Where does it come from?

Reply:
its part of the test - comes from line 701